Mathlib/CategoryTheory/Galois/Examples.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.CategoryTheory.Galois.Basic
+import Mathlib.RepresentationTheory.Action.Basic
+import Mathlib.RepresentationTheory.Action.Limits
+import Mathlib.CategoryTheory.Limits.FintypeCat
+import Mathlib.CategoryTheory.Limits.Shapes.Types
+import Mathlib.Logic.Equiv.TransferInstance
+
+/-!
+# Examples of Galois categories and fibre functors
+
+We show that for a group `G` the category of finite `G`-sets is a `PreGaloisCategory` and that the
+forgetful functor to `FintypeCat` is a `FibreFunctor`.
+
+## Todo
+
+* Characterize connected objects in the category of finite `G`-sets as those with transitive
+  `G`-action
+
+-/
+
+universe u v w
+
+namespace CategoryTheory
+
+namespace PreGaloisCategory
+
+open Limits Functor
+
+private noncomputable def FintypeCat.imageComplement {X Y : FintypeCat.{u}} (f : X ⟶ Y) :
```

Comment:
I believe that the definitions should be in the `FintypeCat` namespace (not `PreGaloisCategory`).